frontend/public/kubevirt/components/vm.jsx

```diff
@@ -330,6 +230,56 @@ const mapDispatchToProps = () => ({
   watchK8sList: actions.watchK8sList,
 });
 
+const ConnectedVmDetails = ({ obj: vm }) => {
+  const vmResource = getResourceKind(VirtualMachineModel, vm.metadata.name, true, vm.metadata.namespace, false);
+  const vmiResources = getResourceKind(VirtualMachineInstanceModel, vm.metadata.name, true, vm.metadata.namespace, true, getLabelMatcher(vm));
```

Comment:
we know vmi name so the getResourceKind function should look like `getResourceKind(VirtualMachineInstanceModel, vm.metadata.name, true, vm.metadata.namespace, false;` - false at the end indicates that we want to watch for single resource, not list